docker/1.4.1/py3/Dockerfile.eia

```diff
@@ -1,22 +1,56 @@
-FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py3.6
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
 
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
+ARG MMS_VERSION=1.0.5
+ARG PYTHON=python3
+ARG PIP=pip3
```

Comment:
Pip should be determined by the python version

Reply:
Removed this ARG